vividus-tests/src/main/resources/story/integration/JbehaveCore.story

```diff
@@ -0,0 +1,5 @@
+Description: Integration tests for JBehave core.
+
```

Comment:
add meta with `@epic` and `@feature` on story-level:
```
Meta:
    @epic vividus-bdd-engine
    @feature composite-steps
```

Reply:
fixed